decidim-surveys/app/views/decidim/surveys/admin/surveys/_question.html.erb

```diff
@@ -43,11 +43,11 @@
     <%= hidden_field_tag "survey[questions][][deleted]", false, disabled: disabled_for_question(survey, question) %>
 
     <div class="survey-question-answer-options">
-      <% unless survey.published? %>
+      <% unless survey.answered? %>
         <button class="button add-answer-option"><%= t('.add_answer_option') %></button>
       <% end %>
 
-      <div class="survey-question-answer-options-list <%= 'published' if survey.published? %>">
+      <div class="survey-question-answer-options-list <%= 'answered' if survey.answered? %>">
```

Comment:
I'm not sure about that, the survey itself is editable. What about `questions_editable?`